### PR TITLE
E2E to list many inactive users in system console

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
E2E to list all inactive users at System console - users management page

#### Ticket Link
Jira ticket: [MM-18512](https://mattermost.atlassian.net/browse/MM-18512)

#### Related Pull Requests
- Has server changes (https://github.com/mattermost/mattermost-server/pull/12772)

### CODE REVIEW

**Comment by saturninoabril** [comment]
Changed reviewer since JE might not be available.

File: e2e/cypress/integration/system_console/inactive_users_spec.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
```
Comment (srkgupta) [inline e2e/cypress/integration/system_console/inactive_users_spec.js]: Can we add a pre-requisite to this test to ensure that default users are created in the system using the command:
./mattermost sampledata -w 4 -u 60 --deactivated-users 200

Comment (saturninoabril) [reply e2e/cypress/integration/system_console/inactive_users_spec.js]: Sure, I'll add that

Comment (saturninoabril) [reply e2e/cypress/integration/system_console/inactive_users_spec.js]: Now updated, PTAL, thanks

**Pull request merged**